marquez_airflow/extractors/bigquery_extractor.py

```diff
@@ -56,12 +56,13 @@ def extract(self) -> [StepMetadata]:
             }
         )]
 
-    def extract_on_complete(self) -> [StepMetadata]:
+    def extract_on_complete(self, task_instance) -> [StepMetadata]:
         steps_meta = self.extract()
 
         client = bigquery.Client()
 
-        job_name = get_job_name(task=self.operator)
+        # job_name = get_job_name(task=self.operator)
+        job_name = task_instance.job_id
```

Comment:
maybe call it bigquery_job_id here to avoid confusion

Reply:
done.